src/main/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+spring.datasource.url=jdbc:postgresql://localhost:5432/Training
+spring.datasource.username= mvillalba
+spring.datasource.password= admin
```

Comment:
This is fine for the training purposes, but just as a tip remember that the sensitive information should not be pushed to the repository, it should be set as secrets in the environment for security 😄